src/core/live-announcer/live-announcer.ts

```diff
@@ -0,0 +1,52 @@
+import {Injectable} from 'angular2/core';
+import {DOM} from '../../core/platform/dom/dom_adapter';
+
+@Injectable()
+export class MdLiveAnnouncer {
+
+  private announcerEl: Element;
+
+  constructor() {
+    this.announcerEl = this._getAnnouncerElement();
+  }
+
+  /**
+   * @param message Message to be announced to the screenreader
+   * @param politeness The politeness of the announcer element.
+   */
+  announce(message: string, politeness = 'polite'): void {
```

Comment:
Cool TypeScript trick: you can introduce your own type for the politeness

```
// higher up
export type AriaLivePoliteness: 'off' | 'polite' | 'assertive';

// then, here
 announce(message: string, politeness: AriaLivePoliteness = 'polite'): void
```
This enforces that people only provide a valid politeness